### ISSUE TITLE
How to get page content ?

### ISSUE THREAD

**Issue by Awilum** [mariosimao/notion-sdk-php#106]
I have page content, but I get properties only 

```
$page = $notion->pages()->find('f3b4ab7c34fa4b71acb0c7fa011374d9');

dd($page);
```

<img width="1270" alt="Screenshot 2022-08-19 at 21 01 59" src="https://user-images.githubusercontent.com/477114/185679974-712aaa55-47e6-4aad-8444-2b2e946d4a94.png">

<img width="551" alt="Screenshot 2022-08-19 at 21 01 44" src="https://user-images.githubusercontent.com/477114/185679971-88b9adf3-46b9-4fdc-b9ce-c8679165df20.png">

**Comment by mariosimao** [comment]
In Notion, pages behave like blocks where the content is their children.

You can fetch page content with
- `$notion->blocks()->findChildren($pageId)` or
- `$notion->blocks()->findChildrenRecursive($pageId)`

The second option iterates over the content to find also their children (useful for nested content).

```php
<?php

require "vendor/autoload.php";

$token = "your_secret_here";
$notion = \Notion\Notion::create($token);

$pageId = "471373adacbe4247aa4b2ce06ed14026";
$content = $notion->blocks()->findChildren($pageId);

dd($content);
```

**Comment by mariosimao** [comment]
Documentation is available on v0.6.2

https://mario.engineering/notion-sdk-php/how-to/find-a-page.html#get-page-content
